src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelectorAll('tr');
+
+for (const row of table) {
+  const cell = document.createElement('th');
```

Comment:
you shouldn't use `th` for all of them. that's why all rows are highlighted

Reply:
I have done)